src/storage/src/render.rs

```diff
@@ -249,6 +250,13 @@ pub fn build_ingestion_dataflow<A: Allocate>(
             let (feedback_handle, feedback) = mz_scope.feedback(Default::default());
 
             let connection = description.desc.connection.clone();
+            tracing::info!(
+                %primary_source_id,
```

Comment:
can we call this field `id` to match many of the other logs emitted from sources? will make it much easier to query logs for a specific source in grafana

Reply:
I had no idea that these fields are stored in a structure way somewhere!